lib/agendash.js

```diff
@@ -163,6 +163,8 @@ module.exports = function (agenda, options) {
     var postMatch = {}
     if (state) postMatch[state] = true
 
+    var limit = 200 // todo UI param
```

Comment:
lets add skip as well, so that can be user for pagination later.  Ex:

skip 0 limit 200 = 1st page
skip 200 limit 200 = 2nd page
skip 400 limit 200 = 3rd page. 

We just need to wire from UI controls.